### PR TITLE
switch types to uint32_t where appropriate

### PR DESCRIPTION
**Pull request by mkv-vcm**

### CODE REVIEW

File: src/securityPolicy.c [hunk 1]
```diff
@@ -296,7 +296,7 @@ security_policy_t policyForSignTxInput()
 // For each transaction (third-party) address output
 security_policy_t policyForSignTxOutputAddressBytes(
         sign_tx_signingmode_t txSigningMode,
-        const uint8_t* rawAddressBuffer, size_t rawAddressSize,
```
Comment (janmazak) [inline src/securityPolicy.c:299]: I think we use size_t everywhere for buffer size, so I'd keep it here, too.

File: src/securityPolicy.h [hunk 2]
```diff
@@ -27,20 +27,20 @@ security_policy_t policyForReturnDeriveAddress(const addressParams_t* addressPar
...
+        uint32_t numWithdrawals,
         bool includeMint
 );
 
 security_policy_t policyForSignTxInput();
 
 security_policy_t policyForSignTxOutputAddressBytes(
         sign_tx_signingmode_t txSigningMode,
-        const uint8_t* rawAddressBuffer, size_t rawAddressSize,
+        const uint8_t* rawAddressBuffer, uint32_t rawAddressSize,
```
Comment (janmazak) [inline src/securityPolicy.h:43]: Let's keep size_t.

File: src/securityPolicy.h [hunk 3]
```diff
@@ -17,7 +17,7 @@ bool is_tx_network_verifiable(
```
Comment (janmazak) [inline src/securityPolicy.h:17]: There's uint16 above, 2x.

Comment (janmazak) [reply src/securityPolicy.h:17]: Also in securityPolicy.c.

**Review by janmazak — CHANGES_REQUESTED** [review]

File: src/securityPolicy.h (hunk 3 above)
Comment (mkv-vcm) [reply src/securityPolicy.h:17]: Done.

File: src/securityPolicy.h (hunk 2 above)
Comment (mkv-vcm) [reply src/securityPolicy.h:43]: Done.

File: src/securityPolicy.c (hunk 1 above)
Comment (mkv-vcm) [reply src/securityPolicy.c:299]: Done.

**Review by janmazak — APPROVED** [review]
Please merge it.

**Pull request merged**